### PR TITLE
create new EC web ui screenshot .md file

### PR DESCRIPTION
**Pull request by vidya22022**
created new EC WEB-UI screenshot .md file

### CODE REVIEW

**Comment by ayasuda-ge** [comment]
could you check-in the screenshots/images? @vidya22022 @vidya2205 @Subhash-vishwakarma

**Review by ayasuda-ge — CHANGES_REQUESTED** [review]
- please check-in the screenshots.
- refer the screenshots in README.
@vidya22022 @vidya2205

File: docs/EC-UI-screenshot.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
+# EC WEB-UI Screenshots as per Tabs
+
+
+## 1)View:
+
+1.1)View: Topology View Table
+
+![image](https://user-images.githubusercontent.com/76679290/109985830-30292280-7d2b-11eb-8dbc-78e256fed96b.png)
```
Comment (ayasuda-ge) [inline docs/EC-UI-screenshot.md]: please update the image link to the relative path. 

https://stackoverflow.com/questions/10189356/how-to-add-screenshot-to-readmes-in-github-repository

Comment (ayasuda-ge) [reply docs/EC-UI-screenshot.md]: @vidya22022 @Subhash-vishwakarma

Comment (ayasuda-ge) [reply docs/EC-UI-screenshot.md]: *image links

**Review by ayasuda-ge — CHANGES_REQUESTED** [review]
please view the comment for the update.

File: docs/EC-UI-screenshot.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+![Alt text](/docs/Ec%20Screenshot/View/ViewTreelist.png?raw=true "Optional Title")
+
+
+
+
+
+## 2)Maintain:
+
+2.1)Maintain: Subscription ->Create
```
Comment (ayasuda-ge) [inline docs/EC-UI-screenshot.md]: Please use proper markdown indenting.

Comment (ayasuda-ge) [reply docs/EC-UI-screenshot.md]: @Subhash-vishwakarma please assist @vidya22022

Comment (Subhash-vishwakarma) [reply docs/EC-UI-screenshot.md]: @ayasuda-ge Vidhya raised a new PR using proper markdown indenting. Please have a review and let us know if any further changes required

**Comment by vidya22022** [comment]
added new screenshot for topology and update index @ayasuda2OO3

File: docs/EC-UI-screenshot.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
+# EC WEB-UI Screenshots as per Tabs
+ * [1.View](#1_view)
```
Comment (ayasuda-ge) [inline docs/EC-UI-screenshot.md]: please use proper markdown numeric indent example here-

https://stackoverflow.com/questions/18088955/markdown-continue-numbered-list

@vidya22022 @Subhash-vishwakarma

Comment (Subhash-vishwakarma) [reply docs/EC-UI-screenshot.md]: yeah Sure @ayasuda-ge will try to close this today

Comment (vidya22022) [reply docs/EC-UI-screenshot.md]: using numeric markdown we got this solution only else we can use bullets markdown @ayasuda-ge

Comment (Subhash-vishwakarma) [reply docs/EC-UI-screenshot.md]: @ayasuda-ge  By this solution we mean using of &nbsp; for space there / for new line

Comment (ayasuda-ge) [reply docs/EC-UI-screenshot.md]: Consider this @Subhash-vishwakarma @vidya22022

1. this is the proper mardown
  1.1 another market down
  1.2 next mark down
  1.3 blah blah
2. final markdown
3. summary

Comment (vidya22022) [reply docs/EC-UI-screenshot.md]: @ayasuda-ge  We marked index with above format but in raw we have used & &nbsp for space we also tried with \ but its not working so is that fine to use &nbsp? or we can go with bullets also one more question i have i created 1 KT document in different file do you want me to add on same screenshot file or its keep as it is separate ?

**Review by ayasuda-ge — APPROVED** [review]
Thanks for the contribution. @vidya22022 @Subhash-vishwakarma

**Pull request merged**